config/webpack.config.js

```diff
@@ -23,12 +24,12 @@ const entryFiles = {
   'hca-rjsf': './src/js/hca-rjsf/hca-rjsf-entry.jsx',
   'health-records': './src/js/health-records/health-records-entry.jsx',
   messaging: './src/js/messaging/messaging-entry.jsx',
+  pensions: './src/js/pensions/pensions-entry.jsx',
+  'post-911-gibs': './src/js/post-911-gibs/post-911-gibs-entry.jsx',
```

Comment:
The only question I have about the name `post-911-gibs` is that we're hiding the word `status`, which is I think the most critical part of the name of this application. Just looking at it, I might not know what `gibs` stands for. But if it was called `post-911-gib-status`, I might guess in context that `gib` is GI Bill, and `status` tells me what this application is used for. Might be overthinking this, but to me it seems a bit clearer that way.

And thanks for alphabetizing these!